apis/apiextensions/v1/composition_environment.go

```diff
@@ -41,11 +41,19 @@ type EnvironmentConfiguration struct {
 // EnvironmentSourceType specifies the way the EnvironmentConfig is selected.
 type EnvironmentSourceType string
 
+// ResolvePolicy is a type for resolve policy.
+type ResolvePolicy string
+
 const (
 	// EnvironmentSourceTypeReference by name.
 	EnvironmentSourceTypeReference EnvironmentSourceType = "Reference"
 	// EnvironmentSourceTypeSelector by labels.
 	EnvironmentSourceTypeSelector EnvironmentSourceType = "Selector"
+
+	// ResolvePolicyAlways is a resolve option.
+	// When the ResolvePolicy is set to ResolvePolicyAlways the reference will
+	// be tried to resolve for every reconcile loop.
+	ResolvePolicyAlways ResolvePolicy = "Always"
```

Comment:
Should there also be a ResolvePolicyOnce value as the default?

Reply:
Please do. I think its a mistake that crossplane-runtime doesn't define it.